website/project/presentation.rst

```diff
@@ -84,175 +90,91 @@ simply working individually.
-present for this Q&A, as many team members as possible should plan to
-attend. Take into account that you may be asked questions that one or
-two specific people in your team are best suited to answer
-(because they were more involved in the part of your code that a
-question relates to).
 
-Teams in a non-North American timezone who are unable to send a few team
-members to the Q&A meeting should contact the instructor in advance to
-schedule an alternate time for their Q&A.
 
 Scoring
 -------
 
-The presentation will be worth 10 points of your final grade, broken
-down as follows:
-
--  5 points for content/structure of the presentation
--  3 points for the demo
--  2 points for the Q&A
-
-The instructor/TAs will use the following rubric, with discretion to
-add/subtract half a point.
-
-Content and structure
-~~~~~~~~~~~~~~~~~~~~~
-
--  5 points: Perfectly structured, clear from beginning to end
--  4 points: Overall well structured. Only a few minor hiccups and/or
-   aspects that were not entirely clear.
--  2.5 points: Needs improvement. Presentation was poorly structured
-   with multiple aspects that were not clear.
--  1 point: Poorly structured. Presentation does not substantially
-   address the content we requested.
--  0: No presentation was delivered.
-
-For avoidance of doubt, this score is based solely on the content and
-structure of the presentation, and whether it addresses all the
-requirements described earlier. Producing a slick video with fancy
-editing (or not) has no bearing on this score.
-
-.. _demo-1:
-
-Demo
-~~~~
+Since you only have one shot at the report and demo, the scoring
+is largely based on the completeness of your final report and demo.
+i.e., we will be paying more attention to whether you put in a
+good-faith effort to provide the information requested in the report, regardless of
+whether there is room for improvement in the report's style,
+level of detail, etc. Similarly, we will be looking at whether
+you actually wrote a demo that meets the requirements listed above,
+regardless of whether the demo could've showcased more aspects
+of your work, etc.
 
--  3 points: Flawless demo that highlights the team’s work.
--  2 points: Smooth demo with only a few small hiccups.
--  1.5 points: Needs improvement. Demo worked but with major issues.
--  1 point: Very incomplete demo. Demo hints at the work the team did,
-   but doesn’t directly showcase their work.
--  0: No demo was done.
+The following issues will have an effect on your score in the
+final report and demo:
 
-.. _qa-1:
+- Not completing any of the sections in the report, or not
+  providing the information requested for those sections.
+- Writing an excessively informal Design and Implementation
+  section (e.g., just providing a series of bullet points)
+- Not doing a live demo at all
+- Doing a live demo that does not fulfil the requirements listed
+  for the live demo.
 
-Q&A
-~~~
+TODO: More concrete specs?
```

Comment:
I feel like it's difficult to be much more specific, but I think we should specify here that it's graded on an ESNU scale.

Reply:
Agree about ENSU grading, some other rubric items might be low-effort/quality in report or the live demo not working?

Reply:
I think this looks good, and I do agree that we should reassure that a demo that fails does not immediately penalize them, but rather in the case of a failed demo we will look at how they respond/adapt to such a situation to still showcase what they have done.

Reply:
Should we also add a bullet point to the above where the Design and Implementation section is left too vague without reference to specific parts of the code or functionalities? Or would such a bullet point be too general and hard to assess concretely?

Reply:
Regarding the live demo, I did indeed have some text on this from the pre-pandemic presentations, and I've added a section on "Live Demo Tips": https://github.com/uchicago-cs/cmsc22000/pull/56/commits/8846c838de7ee116fd97cbb705119255a599f7be

Reply:
Regarding the Design and Implementation section, not sure we want to tell people to avoid being "too vague" without quantifying it more precisely. That said, I've added a small note in the Deisgn and Implementation section telling them that it's a good idea to refer to specific parts of the code when needed: https://github.com/uchicago-cs/cmsc22000/pull/56/commits/cfe2165bf50e7ee6056e60e0cc48b154f1cbf5f9